### DISCUSSION TITLE
Can the cli tool be used with a hosted remote server?

### DISCUSSION THREAD

**Discussion by Dragonatorul** [ArchiveBox/ArchiveBox#1118] [General]
Say I host an ArchiveBox server and I use the browser extension to save to it. At the same time I also want to use the CLI for automation, but it should save to the same remote server, not a local instance of ArchiveBox. Is that possible? It isn't very clear from the cli documentation.

**Comment by pirate** [comment]
Nope, maybe possible in the future but for now just use SSH.

Follow this issue for progress: `Feature Request: Allow locally run ArchiveBox CLI commands to control a separate remote ArchiveBox backend` https://github.com/ArchiveBox/ArchiveBox/issues/786
